package.json

```diff
@@ -49,6 +51,7 @@
     "isparta": "^4.0.0",
     "istanbul": "^0.4.5",
     "mocha": "^3.4.2",
+    "prettier": "^1.7.2",
```

Comment:
Move to `dependencies`

Reply:
Why?

Reply:
So consumers don't have to install `prettier`.

Reply:
I just worry about it needing to be aligned across repos, technically they would need to install it now in order to run the command alongside the linter

Reply:
Spoke offline with Chris. We're going to add prettier as an optional dependency since prettier is only required when extending the prettier config.